msteams-platform/m365-apps/prerequisites.md

```diff
@@ -91,18 +91,37 @@ To install Microsoft 365 Beta Channel applications in your test environment:
 
 ### Mobile
 
-Install the Microsoft 365 app to preview your Teams personal tab app running your physical Android device or Android emulator:
+You can preview Teams personal tabs running in Microsoft 365 app for Android by joining the beta program.
```

Comment:
Let's revert this line (rollout of M365 GA support is stalled at the moment but close to being out).